common/src/main/java/host/bloom/ab/common/commands/sub/Reload.java

```diff
@@ -0,0 +1,21 @@
+package host.bloom.ab.common.commands.sub;
+
+import host.bloom.ab.common.commands.Sender;
+import host.bloom.ab.common.commands.SubCommand;
+import host.bloom.ab.common.config.enums.Messages;
+import host.bloom.ab.common.managers.ConfigManager;
+
+public class Reload implements SubCommand {
+
+    @Override
+    public String getPermission() {
+        return "bab.reload";
```

Comment:
We used this format for the other ones
```suggestion
        return "bab.admin.reload";
```